internal/metrics/collectors/manager.go

```diff
@@ -21,10 +30,12 @@ type LocalManagerMetricsCollector struct {
 	reloadsError     prometheus.Counter
 	lastReloadStatus prometheus.Gauge
 	lastReloadTime   prometheus.Gauge
+	processTotal     *prometheus.GaugeVec
 }
 
 // NewLocalManagerMetricsCollector creates a new LocalManagerMetricsCollector
 func NewLocalManagerMetricsCollector(constLabels map[string]string) *LocalManagerMetricsCollector {
+	labelNames := []string{"generations"}
```

Comment:
Per definition of the task:
```suggestion
	labelNames := []string{"generation"}
```